simapp/app.go

```diff
@@ -362,6 +370,8 @@ func NewSimApp(
 	fswapConfig := fswaptypes.DefaultConfig()
 	app.FswapKeeper = fswapkeeper.NewKeeper(appCodec, keys[fswaptypes.StoreKey], app.AccountKeeper, app.BankKeeper, fswapConfig)
 
+	app.FbridgeKeeper = fbridgekeeper.NewKeeper(appCodec, keys[fbridgetypes.StoreKey], app.AccountKeeper, app.BankKeeper, "stake", authtypes.NewModuleAddress(govtypes.ModuleName).String())
```

Comment:
`stake`? not `cony`? 

Additionally, if the denom is fixed with only one, can it pass only the number without the denom postfix?